src/app/Navbar.module.css

```diff
@@ -0,0 +1,44 @@
+.navbar {
+  /* style */
+  background: rgba(0, 0, 0, 0.2);
+  /* content */

```

Comment:
I have a tendency to add comments to my CSS files to organize my CSS properties. Let me know if this is an issue.